diff -uraN roc-toolkit-0.2.3/src/SConscript omv-roc-toolkit-0.2.3/src/SConscript
--- roc-toolkit-0.2.3/src/SConscript	2023-03-09 11:53:25.000000000 +0100
+++ omv-roc-toolkit-0.2.3/src/SConscript	2023-03-12 17:55:38.580821618 +0100
@@ -141,7 +141,7 @@
                 desc='Real-time audio streaming over the network.',
                 url='https://roc-streaming.org',
                 version=env['ROC_VERSION'])
-            env.AddDistFile(env['ROC_SYSTEM_PCDIR'], pc_file)
+            env.AddDistFile(env['ROC_SYSTEM_LIBDIR'] + '/pkgconfig', pc_file)
 
 if GetOption('enable_examples'):
     examples_env = subenvs.examples.DeepClone()
