diff -up roc-toolkit-0.2.1/src/internal_modules/roc_core/target_libunwind/roc_core/backtrace.cpp.1~ roc-toolkit-0.2.1/src/internal_modules/roc_core/target_libunwind/roc_core/backtrace.cpp
--- roc-toolkit-0.2.1/src/internal_modules/roc_core/target_libunwind/roc_core/backtrace.cpp.1~	2023-01-01 21:12:38.145032822 +0100
+++ roc-toolkit-0.2.1/src/internal_modules/roc_core/target_libunwind/roc_core/backtrace.cpp	2023-01-01 21:19:10.635435557 +0100
@@ -15,6 +15,7 @@
 #define alignas(x) ROC_ATTR_ALIGNED(x)
 #endif
 
+#define alignas _Alignas // make the libunwind/C++ combo great again
 #include <libunwind.h>
 
 #include <stdio.h>
